qfieldsync/gui/package_dialog.py

```diff
@@ -189,6 +202,7 @@ def package_project(self):
         try:
             QApplication.setOverrideCursor(Qt.WaitCursor)
             offline_convertor.convert()
+            open_project(self.original_project_path)
```

Comment:
I think this is already handled internally in the `OfflineConverter`. It should reopen the original project internally, no? If yes, then we need to make sure the OfflineConverter opens the right project.

Reply:
This is done in this [PR](https://github.com/opengisch/libqfieldsync/pull/94)